app/scripts/modules/titus/src/instance/details/instanceDetails.html

```diff
@@ -166,6 +166,10 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
         </dd>
         <dt>Host IP</dt>
         <dd>{{instance.placement.host}}</dd>
+        <dt ng-if="instance.ipv6Addresses.length">IPv6</dt>
```

Comment:
Would it make sense to restructure this DNS section slightly now that there's technically multiple container IPs? i.e. the structure right now is:
- DNS
  - Container IP
  - Host IP
  - IPv6

What if it became:
- DNS
  - Container IPs
    - IPv4
    - IPv6
  - Host IP

Not suggesting a specific visual treatment for now, just asking about taxonomy.

Reply:
I will restructure a bit:
- Container IPv4
- Container IPv6
- Host IP

I want to keep the heading separate since they are separate concepts.

Reply:
👍 seems very reasonable